CHANGELOG.md

```diff
@@ -76,6 +76,7 @@
 - Deployed `MerkleOrchard` V2 to Mainnet, Goerli, Arbitrum and Polygon.
 - Deployed `SingleRecipientGaugeFactory` V2 to Mainnet and Goerli.
 - Deployed `ProtocolIdRegistry` to Mainnet, Goerli, Arbitrum, Polygon, Optimism and Gnosis.
+- Deployed `GaugeWorkingBalanceHelper` to all networks.
```

Comment:
all networks but BSC? :upside_down_face: 
I think the clarification is not super necessary in this case.

Reply:
It's "all networks" that deployed a `VotingEscrowDelegationProxy` :)

I think it's fine. Close enough for governance work! There are plenty of others with exceptions, plus the definition of "all" networks changes over time anyway as we add new ones.

The general policy is to call out when something is "mainnet only" or "L2 only", or applies to a specific and limited set of networks. People can always look at the output to see exactly where it went.